cnesreport.properties

```diff
@@ -1,17 +1,24 @@
 category=Visualization/Reporting
 description=CNES plugin that allows users to download a bundle of project reports in multiple formats.
 homepageUrl=https://github.com/cnescatlab/sonar-cnes-report
-archivedVersions=3.1.0,3.2.2
-publicVersions=3.3.1
+archivedVersions=3.1.0,3.2.2,3.3.1
+publicVersions=4.0.0
 
 defaults.mavenGroupId=fr.cnes.sonarqube.plugins
 defaults.mavenArtifactId=cnesreport
 
+4.0.0.description=Add compatibility with SonarQube 8.9 LTS, enhance reports with new metrics (quality gate status, adherence to coding standard, etc.) and graphs (number of issues, technical debt ratio), and various bugfixes.
+4.0.0.date=2021-09-23
+4.0.0.sqVersions=[8.9]
```

Comment:
If you really want to declare compatibility only with a single version then you don't need range syntax (`[]`), altho it doesn't break anything. But do you really mean to say this isn't compatible with 9.0, 9.1 &etc.?

Reply:
We did not work with SQ 9.0/9.1, only with LTS versions for the sake of simplicity.
As stated in the description, we rely on SQ API and they may evolve on non-LTS releases.
It may actually work for 9.0 or 9.1, but by indicating so, people will ask us for a quick 9.2 compatibility, and so on...
I'm afraid we won't have enough free time to update the plugin on a regular basis.